### ISSUE TITLE
[BUG] APK install failure due to & in filename

### ISSUE THREAD

**Issue by penguinpages** [infinitepower18/WSA-Sideloader#69]
### Describe the bug

Attempting to download and install  IFTTT software to work on some home automation. 
Platform: Windows 11
I have four other Android Apps with IOT devices controlling fine so this is app specific.

Figured I would make a posting to see if this helps.  The output is a bit lighton debug / logs / enabling higher log level

Source for APK
[https://m.apkpure.com/ifttt-automation-workflow/com.ifttt.ifttt/download?utm_content=1008](https://m.apkpure.com/ifttt-automation-workflow/com.ifttt.ifttt/download?utm_content=1008)

```
Error

Unable to install the application. Please check ...blah blah..

[Error Info]
Performing streamed install
operable program or batch file.
```

Questions:
1) within WSA-Sideloader -> can we run this via Powershell and set / increase logging from build and container?
2) Has anyone else tried  IFTTT and can provide feedback on there experience

I perused the "developer options"  such as Select Debug App but .. no Debug App set and no means to set.  

### Steps to reproduce

1) Launch WSA Sideloader
2) Select IFTTT - automation & workflow_4.38.3_Apkpure.apk
3) Install
4) Get error
5) reach for coffee

### Expected behavior

App loads.. move on to next yak in line to shave.

### Download source

Microsoft Store

### WSA version

2303.40000.5.0

### WSA Sideloader version

Version 10.0.22000.556 (Windows 11)

### Windows build number

22624.1690

**Comment by infinitepower18** [comment]
Hi, just tried it and it seems like the file name is causing this error... I'll look further into this but for now can you rename the file to IFTTT.apk and then installing it?

**Comment by infinitepower18** [comment]
Figured it out, it is due to & being present in the file name. I have found a fix for this issue and will release an update once I fix some bugs related to the partially running mode in WSA. I'm having exams this month so it may take a while.

**Comment by penguinpages** [comment]
The rename of file fixed this and worked fine.   Thanks for response.   Help with "how to debug"  ex: what you did go get to that event error, would help.   And good luck on your exams...

**Comment by infinitepower18** [comment]
This issue has been solved in version 1.3.13 which is now available on the Microsoft Store. Thanks a lot for your support and patience. Do let me know any bugs you encounter in the future.
